make/stop

```diff
@@ -12,5 +12,6 @@ has_namespace() {
 stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-stop start
 if has_namespace ; then
     kubectl delete namespace "${NAMESPACE}"
+    helm delete $(helm list --namespace  "${NAMESPACE}"|grep -v NAME| awk '{ print $1 }')
```

Comment:
This would be a lot easier if we just added `--name=${NAMESPACE}` to `make/run`.
Otherwise, use `helm list | awk " \$NF == ${NAMESPACE} { print \$1 }"` maybe?

Reply:
You mean in the Makefile ?
I actually can simplify this now that I know about `-q` for helm list.

Reply:
Done with commit 25e5e602dcd288ff89137c946d06066bba931000